### ISSUE TITLE
Help: ... Failure (bad HTTP status: 404: Error 404 : Page not found )

### ISSUE THREAD

**Issue by DevXen** [qdm12/ddns-updater#235]
**TLDR**: FreeDNS is not connecting. it says: Failure (bad HTTP status: 404: Error 404 : Page not found ), 1m48s ago

1. Is this urgent: No
2. DNS provider(s) you use: FreeDNS/DuckDNS
3. Program version: Not sure how to check version on unraid (no logs in appdata, only config/updates json files). but I am using the latest docker image from repository: qmcgaw/ddns-updater. also it only appeared a few weeks ago in the community apps so i'd guess it's on the latest. 

    

    `Running version latest built on 2020-03-13T01:30:06Z (commit d0f678c)`

4. What are you using to run the container: docker-compose? Unraid Docker.
5. Extra information (optional)

Logs: 

```
2021/08/22 08:26:48 INFO Updating record [domain: homeplex.org | host: [removed] | provider: freedns | ip: ipv4] to use [IP removed]
2021/08/22 08:26:48 ERROR bad HTTP status: 404: Error 404 : Page not found
```

Configuration file (**remove your credentials!**):

```json
{
  "settings": {
      "provider": "freedns",
      "domain": "homeplex.org",
      "host": "[Removed]",
      "token": "Token from afraid.org - found in the link to directly update IP",
      "ip_version": "ipv4"
  }
}
```

it's worth noting that none of my FreeDNS addresses update. but all my DuckDNS ones do.

Host OS: Unraid

**Comment by qdm12** [comment]
Can you try `https://sync.afraid.org/u/<your-token>/` in your browser (make sure to remove the url from your history to not leak your token)? What do you get?

**Comment by jhotmann** [comment]
Just confirming that all my FreeDNS update URLS are https://freedns.afraid.org/dynamic/update.php?token

**Comment by qdm12** [comment]
@DevXen sorry for the delay, I must had missed the notification email.

@jhotmann Nice! But should the program use the php url instead to simplify setup? Or do you see any reason to prefer the v2 dynamic dns url?

**Comment by DevXen** [comment]
@jhotmann That worked. thank you for your help. I agree the documentation should note that.

@qdm12 it's all good and i got it working. thank you for your help as well.

**Comment by qdm12** [comment]
(Sorry to repeat)

@jhotmann Should the program use the php url instead to simplify setup? Or do you see any reason to prefer the v2 dynamic dns url?

**Comment by jhotmann** [comment]
> (Sorry to repeat)
> 
> @jhotmann Should the program use the php url instead to simplify setup? Or do you see any reason to prefer the v2 dynamic dns url?

I would stick with the current setup. Their V2 API has some advantages over V1 and with the updated documentation it shouldn't be an issue to configure.
